batch/deployment.yaml

```diff
@@ -34,6 +34,26 @@ spec:
          value: "spot"
 {% endif %}
       containers:
+      - name: nginx
+        image: {{ batch_driver_nginx_image.image }}
+        resources:
+          requests:
+            cpu: "4"
+            memory: "2G"
+          limits:
+            cpu: "4.5"
+            memory: "4G"
```

Comment:
This is gonna increase our testing burden a fair bit. I guess that's what needs to be done to get performance, but we should go into this eyes open.

Reply:
Ya I was thinking about this, and contemplated if we should run smaller PR deployments, but I'm not sure I see a way to drive test times down if we don't give the resources we can to PR drivers.